modules/foreman_ptable.py

```diff
@@ -351,7 +357,7 @@ def main():
         if state == 'present_with_defaults':
             module.fail_json(msg="'state: present_with_defaults' and 'name: *' cannot be used together")
         if state == 'absent':
-            if entity_dict.keys() != ['name']:
+            if entity_dict.keys() != ['name', 'locked']:
```

Comment:
What is the reasoning here?
I think, it breaks exactly, what the `if` was meant for.

Reply:
It doesn't make sense to provide `name` or `locked` if you're just removing the ptable (state = absent). Right?

Reply:
Oh, nevermind :) Yeah, this shouldn't be changed. It's saying that `name` is the *only* valid option when state = absent.